DigitalLearningSolutions.Web/Helpers/CompletionSummaryHelper.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System;
+
+    public static class CompletionSummaryHelper
+    {
+        public static string GetCompletionSummary(
+            DateTime? completed,
+            int maxPostLearningAssessmentAttempts,
+            bool isAssessed,
+            int postLearningAssessmentPassThreshold,
+            int diagnosticAssessmentCompletionThreshold,
+            int tutorialsCompletionThreshold
+        )
+        {
+            if (completed != null)
+            {
+                return $"You completed this course on {completed:dd MMMM yyyy}.";
+            }
+
+            if (isAssessed)
+            {
+                if (maxPostLearningAssessmentAttempts == 0)

```

Comment:
I'd prefer to avoid having this nested if, either by using a ternary operator, or by combining it with the if above 🙂

Reply:
Yep, changed in 53a4d70 🙂